File: aida-cli-lib/src/tests/fixtures/criteria/sample.py
# trace:STORY-1.A1 | ai:codex
def test_python_above():
    pass

class TestGroup:
    def test_python_inside(self):
        # trace:STORY-1.A2 | ai:codex
        assert True

def test_python_untraced():
    pass

File: aida-cli-lib/src/tests/fixtures/criteria/sample.ts
// trace:STORY-1.A1 | ai:codex
test("typescript above", () => {
  expect(true).toBe(true);
});

it('typescript inside', () => {
  // trace:STORY-1.A2 | ai:codex
  expect(true).toBe(true);
});

test("typescript untraced", () => {
  expect(true).toBe(true);
});
